pkg/apiserver/info/info.go

```diff
@@ -73,6 +77,8 @@ func (s *Service) infoHandler(c *gin.Context) {
 		Version:            version.GetInfo(),
 		EnableTelemetry:    s.params.Config.EnableTelemetry,
 		EnableExperimental: s.params.Config.EnableExperimental,
+		EnableConprof:      conprof.IsFeatureEnable(s.params.Config),
+		EnableNonRootLogin: nonrootlogin.IsFeatureEnable(s.params.Config),
```

Comment:
How about call them support instead of enable? "Enable" sounds like they can be enabled. However in fact they cannot.

For example:

```js
SupportConprof: ...
SupportNonRootLogin: ...
```

Also, considering that we may have more things to add in the support matrix, can we achieve this?

```js
SupportedFeatures: ["conprof", "nonRootLogin", ...]
```

Reply:
OK